pkgs/by-name/te/tenv/package.nix

```
@@ -0,0 +1,27 @@
+{ buildGoModule, lib, fetchFromGitHub }:
+buildGoModule rec {
+  pname = "tenv";
+  version = "v1.2.0";
+
+  src = fetchFromGitHub {
+    owner = "tofuutils";
+    repo = "tenv";
+    rev = version;
+    sha256 = "sha256-yLNdBwKF6Jts661P+YZhFGNr71TG7Scb6RGvFxTLqYQ=";
+  };
+
+  vendorHash = "sha256-GGWiP1rIDF6qxST2ZmnKNkgbS+15hxaCs1d1+UEiYgU=";
+
+  # Disable tests since it requires network access and relies on the
+  # presence of release.hashicorp.com
+  doCheck = false;
+
+  meta = with lib; {
+    description =
+      "OpenTofu, Terraform and Terragrunt version manager, written in Go.";
+    mainProgram = "tenv";
+    homepage = "https://github.com/tofuutils/tenv";
+    license = licenses.apache_2_0;
+    maintainers = with maintainers; [ rmgpinto ];
+  };
```

```suggestion
  meta = {
    description = "A version manager for OpenTofu, Terraform and Terragrunt written in Go";
    homepage = "https://github.com/tofuutils/tenv";
    license = lib.licenses.asl20;
    maintainers = with lib.maintainers; [ rmgpinto ];
  };
```
Avoid using `with`.
Feel free to improve on description.

mainProgram is unnecessary when name equates package name. Test running  with `nix run .#tenv` (I have not tested)